client/src/cmdlfhid.c

```diff
@@ -630,8 +656,8 @@ static command_t CommandTable[] = {
     {"reader",  CmdHIDReader,   IfPm3Lf,         "attempt to read and extract tag data"},
     {"clone",   CmdHIDClone,    IfPm3Lf,         "clone HID tag to T55x7"},
     {"sim",     CmdHIDSim,      IfPm3Lf,         "simulate HID tag"},
-    {"brute",   CmdHIDBrute,    IfPm3Lf,         "bruteforce card number against reader"},
-    {"watch",   CmdHIDWatch,    IfPm3Lf,         "continuously watch for cards.  Reader mode"},
+    {"brute",   CmdHIDBrute,    IfPm3Lf,         "bruteforce HID tag while maintaining specific parameter values"},
+    {"watch",   CmdHIDWatch,    IfPm3Lf,         "continuously watch for cards"},
```

Comment:
No need to change text

Reply:
I've reverted the removal of "Reader Mode", but I still think the change to line 633 is necessary as part of the PR (text updated).